components/eam/src/chemistry/mozart/lin_strat_chem.F90

```diff
@@ -188,6 +188,18 @@ subroutine lin_strat_chem_inti(phys_state)
 !     
 !   initialize the linoz data
 
+    !!set true for linoz_v3 interpolation in interpolation
+    if (linoz_v3) file%linoz_v3=.true.
+    if (linoz_v2) file%linoz_v2=.true.
+    !!
+    if (linoz_v2.eq..true. .and.linoz_v3.eq..true. ) then
+            write(iulog,*) subname // ', linoz: Both linoz_v2 and linoz_v3 are true, please check'
+            return
+    elseif (linoz_v2.eq..false..and.linoz_v3.eq..false.) then
+            write(iulog,*) subname // ', linoz: Both linoz_v2 and linoz_v3 are false, please check'
```

Comment:
Both false is certainly possible. I'd revise the message to "..., please check these settings are correct" and let the model continue to run.

Reply:
How about this?
if (linoz_v2.eq..true. .and.linoz_v3.eq..true. ) then
            write(iulog,*) 'linoz_readnl, linoz: Both linoz_v2 and linoz_v3 are true, please check if these settings are correct'
             return
    elseif (linoz_v2.eq..false..and.linoz_v3.eq..false.) then
            write(iulog,*) 'linoz_readnl, linoz: Both linoz_v2 and linoz_v3 are false, please check if these settings are correct'
    endif

Reply:
First, confirm `return` stops the model. Otherwise, revise it.

Second, make the message clearer (something like):
```
if (linoz_v2.eq..true. .and.linoz_v3.eq..true. ) then
write(iulog,*) 'linoz_readnl, linoz: Both linoz_v2 and linoz_v3 are true. This is wrong! Please check.'
return
elseif (linoz_v2.eq..false..and.linoz_v3.eq..false.) then
write(iulog,*) 'linoz_readnl, linoz: Both linoz_v2 and linoz_v3 are false. Please check if these settings are correct.'
endif
```

Reply:
Hi Qi. It is tested and return works fine. The code is updated.